UT4MasterServer/Services/MatchmakingWaitTimeEstimateService.cs

```diff
@@ -0,0 +1,55 @@
+using UT4MasterServer.Models.Requests;
+
+namespace UT4MasterServer.Services;
+
+public class MatchmakingWaitTimeEstimateService
+{
+	private Dictionary<string, List<(DateTime DeleteTime, double WaitTime)>> estimates;
+	private static readonly TimeSpan RelevantReportTimeDuration = TimeSpan.FromMinutes(1);
+
+	public MatchmakingWaitTimeEstimateService()
+	{
+		estimates = new Dictionary<string, List<(DateTime, double)>>();
+	}
+
+	public void AddWaitTime(string mode, double seconds)
+	{
+		if (!estimates.ContainsKey(mode))
+			estimates.Add(mode, new List<(DateTime, double)>());
```

Comment:
This is the reason why you should put braces after IFs, this code looks like it is unformatted, instead of being IF block + additional non-related line.